src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector_SequencePoints.cs

```diff
@@ -178,6 +170,8 @@ internal static void GetFirstLocalOrFieldBreakpointSpan(SyntaxToken? firstToken,
             int end;
             if (declarationSyntax.Variables.Count == 1)
             {
+                Debug.Assert(declarationSyntax.Parent != null);
```

Comment:
Should we assert this right after `var declarationSyntax = ...` since we are unconditionally using it in `node = declarationSyntax.Parent!;` at the end of this method anyway?

Reply:
Yes, missed that one.